lib/mem.h

```diff
@@ -74,14 +74,14 @@
 #ifndef MEM_H
 #define MEM_H
 
-// #define MEM_USE_RTL	1
+#define MEM_USE_RTL	1
```

Comment:
Do you think there is a case where we would want to go back? If not, we should remove the define completely. Otherwise make it a CMake option (but I personally think it's unnecessary)

Reply:
This switch was for debugging memory allocations, but it was enabled for some reason. Personally, I don't find any use for this functionality, but who knows, maybe this can be somewhere useful. CMake option is good idea.